tsc/meetings/2020-08-18.md

```diff
@@ -0,0 +1,66 @@
+Minutes from 61st OpenVDB TSC meeting, Aug 4th, 2020, (EDT)
+
+Attendees: *Nick* A., *Jeff* L., *Ken* M.
+
+Additional Attendees: Johannes Meng (Intel), JT Nelson (Blender),
+Bruce Chernia (Intel), Peter Cheng (DW), Andre Pradhana (DW)
+
+Regrets: *Peter* C, *Dan* B..
+
+Agenda:
+
+1) Confirm quorum
+2) Secretary
+3) Forum
+4) NanoVDB
+5) Repository Structure
+6) Makefile
+7) Next Meeting
+
+1) Quorum was confirmed.
+
+2) Secretary was Jeff Lait
+
+3) Forum
+
+Questions about computing averages of leaf nodes.  Unclear if the questioner wants it baked in the tree or just computed.  "On load" implies it is to be computed on construction.
+
+Another post is on large VDBs and Out of Core (OOC).  Wants to be able to stream it and unload it.  Ken has in the past written a minimal version that does delayed loading and unloading.  Loads in only the leaf data into a preset pool.
+
+A mistake we've tried to do is make VDB try to do everything.  So we kept putting everything in one structure, so it is not ideal for any application.  Out of core is substandard, and random access is harmed by the out of core.  We're trying to simplify the tree, and reduce dependency.  Ken would like to contribute his streamable read only tree at some point.  Ken will reply to the thread.
```

Comment:
Sounds good. I think we should try and measure the difference in performance when using out-of-core, I expect you're right that it is not negligible, but would be good to understand precisely what the cost is. I'll try and prioritise adding a CMake flag that allows us to disable it soon.

Reply:
Just a note of caution here. It's surprisingly difficult to evaluation this perform overhead because it depends highly the CPUs ability to do branch-prediction. Good branch-prediction results in a relatively low overhead of having out-of-core (since in most cases data are in fact in-core).